<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtlMovieDvdExtractionWindow</class>
 <widget class="QtlMovieMainWindowBase" name="QtlMovieDvdExtractionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtlMovie DVD Extraction</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QtlMovie.qrc">
    <normaloff>:/images/qtlmovie-64.png</normaloff>:/images/qtlmovie-64.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="layoutCentral">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="box" native="true">
       <layout class="QVBoxLayout" name="layoutBox">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabDvd">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabIso">
           <attribute name="title">
            <string>ISO</string>
           </attribute>
           <layout class="QGridLayout" name="layoutTabIso">
            <item row="1" column="0">
             <widget class="QLabel" name="labelFullPath">
              <property name="text">
               <string>Full path :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelIsoFile">
              <property name="text">
               <string>ISO file name :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QtlLineEdit" name="editIsoFile"/>
            </item>
            <item row="2" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="valueFullPath">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabTitleSets">
           <attribute name="title">
            <string>Video Title Sets</string>
           </attribute>
           <layout class="QVBoxLayout" name="layoutTabTitleSets">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableWidget" name="tableTitleSets">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabFiles">
           <attribute name="title">
            <string>Files</string>
           </attribute>
           <layout class="QVBoxLayout" name="layoutTabFiles">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableWidget" name="tableFiles">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="boxSelection" native="true">
          <layout class="QGridLayout" name="layoutSelection">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="1" column="1">
            <widget class="QLabel" name="labelDvd">
             <property name="text">
              <string>Input DVD :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="comboDvd"/>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="buttonRefreshDvd">
             <property name="text">
              <string>Refresh</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="labelDestination">
             <property name="text">
              <string>Destination directory :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QtlLineEdit" name="editDestination"/>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="buttonBrowseDestination">
             <property name="text">
              <string>Browse ...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="boxProgress" native="true">
          <layout class="QGridLayout" name="layoutProgress">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="1" column="0">
            <widget class="QPushButton" name="buttonStart">
             <property name="text">
              <string>Start ...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>0</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="labelRemainingTime">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QtlPlainTextLogger" name="log">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>629</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="actionStart"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionClearLog"/>
    <addaction name="actionLogDebug"/>
    <addaction name="actionSaveLog"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionReleaseNotes"/>
    <addaction name="actionSearchNewVersion"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About QtlMovie</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start ...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings ...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionClearLog">
   <property name="text">
    <string>Clear Log</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSaveLog">
   <property name="text">
    <string>Save Log ...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help ...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionLogDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log Debug</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSearchNewVersion">
   <property name="text">
    <string>Search New Version</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionReleaseNotes">
   <property name="text">
    <string>Release Notes ...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh ...</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtlMovieMainWindowBase</class>
   <extends>QMainWindow</extends>
   <header>QtlMovieMainWindowBase.h</header>
   <container>1</container>
   <slots>
    <slot>editSettings()</slot>
    <slot>aboutMediaTools()</slot>
    <slot>about()</slot>
    <slot>showHelp()</slot>
    <slot>showReleaseNotes()</slot>
    <slot>searchNewVersion()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>QtlPlainTextLogger</class>
   <extends>QPlainTextEdit</extends>
   <header>QtlPlainTextLogger.h</header>
   <slots>
    <slot>saveToFile()</slot>
    <slot>setDebugMode(bool)</slot>
    <slot>save()</slot>
    <slot>saveToFile(QString)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>QtlLineEdit</class>
   <extends>QLineEdit</extends>
   <header>QtlLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtlMovie.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>QtlMovieDvdExtractionWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>QtlMovieDvdExtractionWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>QtlMovieDvdExtractionWindow</receiver>
   <slot>editSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClearLog</sender>
   <signal>triggered()</signal>
   <receiver>log</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveLog</sender>
   <signal>triggered()</signal>
   <receiver>log</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelp</sender>
   <signal>triggered()</signal>
   <receiver>QtlMovieDvdExtractionWindow</receiver>
   <slot>showHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLogDebug</sender>
   <signal>toggled(bool)</signal>
   <receiver>log</receiver>
   <slot>setDebugMode(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSearchNewVersion</sender>
   <signal>triggered()</signal>
   <receiver>QtlMovieDvdExtractionWindow</receiver>
   <slot>searchNewVersion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReleaseNotes</sender>
   <signal>triggered()</signal>
   <receiver>QtlMovieDvdExtractionWindow</receiver>
   <slot>showReleaseNotes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRefresh</sender>
   <signal>triggered()</signal>
   <receiver>QtlMovieDvdExtractionWindow</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRefreshDvd</sender>
   <signal>clicked()</signal>
   <receiver>QtlMovieDvdExtractionWindow</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editDestination</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QtlMovieDvdExtractionWindow</receiver>
   <slot>updateIsoFullPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editIsoFile</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QtlMovieDvdExtractionWindow</receiver>
   <slot>updateIsoFullPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBrowseDestination</sender>
   <signal>clicked()</signal>
   <receiver>QtlMovieDvdExtractionWindow</receiver>
   <slot>browseDestination()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refresh()</slot>
  <slot>updateIsoFullPath()</slot>
  <slot>browseDestination()</slot>
 </slots>
</ui>
